src/devtools/client/webconsole/actions/input.js

```diff
@@ -113,6 +113,30 @@ function evaluateExpression(expression) {
   };
 }
 
+function eagerEvalExpression(expression) {
```

Comment:
Let's migrate this to .ts file, it won't take too long since it's only a handful of actions.